addon/-private/column-tree.js

```diff
@@ -359,7 +359,7 @@ const ColumnTreeNode = EmberObject.extend({
       delta = Math.max(Math.min(oldWidth + delta, maxWidth), minWidth) - oldWidth;
 
       if (delta === 0) {
-        return;
+        return oldWidth;
```

Comment:
Further down in two places:

```js
return set(column, 'width', oldWidth + delta);
```

Should also be:

```js
let width = oldWidth + delta;
set(column, 'width', width);
return width;
```

Relying on the return value from `set` is a bit weird (and behaves differently in some version of Ember 3.x iirc). Better to just return the value explicitly.

Reply:
👍